src/Host.cpp

```diff
@@ -31,25 +31,15 @@
 #include "TMap.h"
 #include "TRoomDB.h"
 #include "TScript.h"
-#include "XMLexport.h"
 #include "XMLimport.h"
 #include "dlgMapper.h"
 #include "dlgTriggerEditor.h"
 #include "mudlet.h"
 
 #include "pre_guard.h"
 #include <QtUiTools>
-#include <QApplication>
-#include <QDir>
-#include <QDirIterator>
-#include <QMessageBox>
-#include <QStringBuilder>
-#include "post_guard.h"
-
-
-#include <errno.h>
 #include <zip.h>
```

Comment:
Why is this inside the guarded section now?

Reply:
My understanding is that the guards are for mcvs's leak detection to ignore 3rd party libraries - hence I moved it in.

Reply:
It's just a bit a bit inconsistent then. But if you feel it's okay, feel free to merge it in.